index.html

```diff
@@ -6,5 +6,5 @@
     <div id='root'>
     </div>
   </body>
-  <script src="/static/bundle.js"></script>
-</html>
\ No newline at end of file
+  <script src="static/bundle.js"></script>
```

Comment:
The leading `/` is removed in order to allow the built app to be accessed directly from the file system. It will still work when running in dev-server.